apps/zotonic_core/src/install/z_install_update.erl

```diff
@@ -734,8 +735,48 @@ rsc_language(C, Database, Schema) ->
 
             {ok, _} = epgsql:equery(C, "
                             insert into pivot_task_queue (module, function, key)
-                            values ('z_install_update_task', 'init_language', $1)
+                            values ('z_install_update_task', 'init_language', '')
                             ",
-                            [ z_ids:id() ]),
+                            []),
```

Comment:
Will the key be inserted at an other point?

Reply:
There is an unique SQL index per module/function/key.
In this case there can only be a single task for this, so we can leave the key empty.

Reply:
```sql
    CREATE TABLE pivot_task_queue
    (
        id serial NOT NULL,
        module character varying(80) NOT NULL,
        function character varying(64) NOT NULL,
        key character varying(100) NOT NULL DEFAULT ''::character varying,
        due timestamp with time zone ,
        props bytea,
        error_count integer not null default 0,

        CONSTRAINT pivot_task_queue_pkey PRIMARY KEY (id),
        CONSTRAINT pivot_task_queue_module_function_key_key UNIQUE (module, function, key)
    )
```